### PR TITLE
Adds 100% Coverage to Share Function

### PR DESCRIPTION
**Pull request by redreceipt**
## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apolloschurchapp/src/utils/content/__tests__/share.tests.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+jest.mock('Platform');
+jest.mock('Share');
+
+describe('the share function', () => {
+  beforeAll(() => {
+    // we don't need to test the functionality of Share.share(),
+    // only that our args are being sent properly
+    Share.share = jest.fn((args) => {
+      throw new Error(JSON.stringify(args));
```
Comment (conrad-vanl) [inline packages/apolloschurchapp/src/utils/content/__tests__/share.tests.js]: While this is a clever way to test the function, there's actually a simpler way to do this!
Check out - https://jestjs.io/docs/en/mock-functions#custom-matchers

Essentially you could simplify this line to just be `Share.share = jest.fn()` and in your test below, do something like:

```
share({ title: 'title', url: 'http://url.com' });
expect(Share.share).toBeCalledWith(
    expect.objectContaining({
      title: 'title',
      message: 'title',
      url: 'http://url.com',
    }),
  );
```

Comment (redreceipt) [reply packages/apolloschurchapp/src/utils/content/__tests__/share.tests.js]: incredible

**Comment by conrad-vanl** [comment]
> con-ratified it

con- _rad_ ified it 😉
